spec/index.html

```diff
@@ -147,7 +147,7 @@ <h3>Graph-based Data Model</h3>
     <figure id="fig-rdf-graph">
       <a href="rdf-graph.svg">
         <object data="rdf-graph.svg"
-          aria-label="An RDF graph with two nodes (Subject and Object) and a triple connecting them (Predicate)"></object>
+          aria-label="An RDF graph with two nodes (Subject and Object) and an arc connecting them (Predicate)"></object>
       </a>
       <figcaption>An RDF graph with two nodes (Subject and Object) and a triple connecting them (Predicate)</figcaption>
```

Comment:
```suggestion
          aria-label="An RDF graph with two nodes (Subject and Object) and an arc (Predicate) connecting them"></object>
      </a>
      <figcaption>An RDF graph with two nodes (Subject and Object) and an arc (Predicate) connecting them</figcaption>
```

Reply:
oh wow, thanks for spotting that I forgot to change the caption!